### ISSUE TITLE
Health Information Exchange

### ISSUE THREAD

**Issue by pristineliving** [pristineliving/Team-Peony-Primer#7]
## A Recent History of HIE

In order to understand the complex infrastructure and challenges that pervade HIE, it's first important to reflect on the roots of the existing system. Although health information exchange has existed long before electronic records, much of what we understand today as HIE began in the 1990s, in particular with the Hartford Foundation's funding of **"Community Health Management Information Systems"**. These came in the form of grants to 7 locations nationwide, which were tasked with developing centralized data repositories to house important patient information – demographics, clinical data, health insurance information, etc. The central purpose of these systems was actually for assessment and patient billing; to create a database with individual insurance information that was easily accessible and could be verified right away.

As with any pioneering system, the Community Health Management Information Systems were rife with major problems that resulted in all of them being abandoned by the mid '90s. Primarily they were related to the technology still in its infancy: slow internet connections, and issues with data integration. Internet technology was still in its prototypic stage, and this was the first time health organizations endeavored to substantially integrate disparate health information – a task we know is more of an undertaking than many initially believe. Privacy and security concerns will also hot-button issues, and a foundation of grant funding led to issues in transitioning off its support and onto that of other stakeholders.

As the 1990s progressed into the midpoint of the decade, HIE also saw a shift towards more individualized and commercial efforts. Namely, **Community Health Information Networks (CHINs)**. CHINs were built towards the goal of cost reduction through data sharing, with a focus on saving costs associated with moving data between providers. To address the privacy and security concerns of the previous information systems, CHINs adopted a decentralized architecture. However, such measures did not ultimately prevent most networks from failing. There remained a lack of focus on community stakeholders, which fostered a roundtable of competitors – a difficult model of success. Generally, vendors who built these systems also pit the interests of hospitals against the community and left the return on investment vague to many stakeholders who might have otherwise shown an interest.

Then, in the late 90's and early 2000s, the national focus panned for the first time to patient safety and quality of care. It came with the Institute of Medicine's reports on the same topic, first in 1999 with **"To Err is Human"** and again with a follow-up in 2001. These were the first of many reports to ignite a new interest after suggesting that up to 98,000 people died annually as a result of preventable medical errors. the 2001 follow-up focused more on HIT, including HIE, and their potential role in error reduction and improvement in efficiency and effectiveness within the healthcare system. This new focus on quality formed the basis of several new downstream developments of novel HIE systems.

 **Regional Health Information Organizations (RHIOs)** were among the first, coming into development in the early 2000s. Local 'neutral' organizations – generally consisting of representatives from a variety of provider organizations – began bringing providers together to achieve HIE goals, primarily improving the quality of care in communities. The idea behind these organizations was that in order to facilitate successful HIE, everyone at the table needed to overcome challenges within their market, whether political, commercial, financial, regulatory, or legal. Thus, the chief responsibilities of each RHIO were four-fold: bring together stakeholders; manage agreements among them; manage security and their access to information; and manage virtual identities and log-in credentials. RHIOs continue to develop today, each unique in how they approach the challenges of HIE.

New advancements came again in 2004 when the Department of Health and Human Services (DHHS) began developing the **National Health Information Network (NHIN)**. This replaced the DHHS' earlier concept of a National Health Information Infrastructure (NHII) with a 2-phase development system for integrating a prototype architecture, specifications and services, and working constructs.

Shortly after, however, HIE was dealt a shocking blow in 2006 with the closure of what was considered a leading RHIO in Santa Barbara County, CA after 8 years of operations. Reasons for its closure boiled down to its grand funding and small vendor foundations. While the RHIO had a strong leadership by its grant foundation and grantees, it lacked involvement by community stakeholders who failed to move into more prominent leadership roles as expected. Further, the small vendor developing the RHIO from scratch could not keep up with the modern HIE demands. Above all, the failure of this RHIO was a lesson in building trust and the need for active involvement from health players.

Among the most defining features of modern HIE came in 2009, with the **Health Information Technology for Economic and Clinical Health (HITECH) Act**, specifying HIE as a core component in "meaningful use" incentives. This was the broadest federal support for HIE to date and presented on several fronts. HITECH provides incentives for EHR adoption among providers such that they are able to demonstrate they use the EHR systems in a "meaningful" way – the criteria for which includes the ability to exchange health information. HITECH also addressed increasing privacy and confidentiality concerns previously outlined in HIPAA.

As a direct response to the HITECH Act, in 2010 the office of the National Coordinator for Health IT (ONC) publicly assured stakeholders that NHIN would not exchange private health information with government agencies. This assurance came among resurfacing privacy issues and the forward propulsion of HIE systems. With NHIN continuing in development, it was a growing concern that information would be increasingly easy to transmit, and thus government agencies would have access to private and sensitive information. As a response, NHIN adopted stronger privacy protections guided by the regulations outlined by the HITECH act.

As we move forward into the new decade, we have seen that many of these challenges that pervade HIE development still continue today. But, rather than the national network, many organizations set out to establish, several different architectures for HIE have developed and diverged. Today we can see three unique types of HIE: directed exchange, between two or more providers; query-based exchange, where entities may query an HIE system to find specific information on a particular patient; and consumer-mediated exchange, in which the patient themselves is responsible for coordinating information. 

## What Is HIE and How It Works

The healthcare system in the U.S. has been long fragmented while patients are receiving treatment from different providers during episodes of care, possibly referred to or transferred to many other providers (Holmgren, 2017). During the inter- and intra-institutional transferring, different formats of patient information, e.g., faxes, medical records, prescription, vital signs, and the lab measures, could make a difference in the decision making with less incompleteness. After the broad adoption of electronic health records (EHR), it becomes possible for healthcare practitioners to access patient information with more details. As we have discussed in section 1, the framework of Health Information Exchange (HIE) was established to solve the dilemma of incomplete and non-timely patient information delivery.

Health Information Exchange (HIE) is the electronic transfer of patient data and health information between healthcare providers and has become a part of the health care restructuring program to facilitate the sharing of medical information between healthcare service providers (Esmaelzadeh, 2016). HIE is defined as the transfer of electronic health information – laboratory test results, medication lists, and other clinical information – among organizations and providers (Kwan, 2013). A health information exchange (HIE) addresses the need for clinicians to have access to patient information from various sources so that clinical communication and care coordination could be improved during transitions of care (Rosenfeld, 2006). With help from HIE, providers are able to make clinical decisions with less incomplete information with less misdiagnosis, avoiding unsafe or suboptimal treatment, and redundant healthcare services (Holmgren, 2017). The concept of HIE is to make all related patient information accessible to the decision-making providers whenever needed, regardless of who generates the relevant information. And it is perceived that HIE should address the fragmented forms of patient information so that the quality of healthcare, patient satisfaction, and diagnostic efficiency could all be improved with a reduction of cost and unnecessary usage of medical services (Walker, 2005). So far, England, Netherlands, Finland, and the U.S. have adopted and advanced the implementation of HIE regionally and nationally. By 2014, more than 100 organizations facilitate HIE among provider organizations, and 30% of hospitals and 10% of ambulatory clinics participate in the HIE network in the U.S (Adler-Milstein, 2013). 

<img width="1287" alt="Screen Shot 2020-06-15 at 4 53 34 AM" src="https://user-images.githubusercontent.com/15147292/84654563-46a0fc80-aec4-11ea-913c-a138bd5df9b3.png">

**Table 1. Four key dimensions of HIE**

HIE is defined to be “making all information available when needed”. It requests the providers to electronically share the patient profile ranging from a single type such as MRI results or lab test results, a summary of care records, and a complete set of patient records (DesRoches, 2015). The final output delivery depends on who provided the patient profile and who requested the patient information. For instance, the communication between the emergency department (ED) and the primary care provider (PCP) will require that the most up-to-date summary of patient medical history, comorbidity, and vital signs are ready to view at such a time-sensitive circumstance. Yet when the PCP communicates with internal medicine for chronic disease management, the desired form and content of patient information will vary. Although the idealized HIE provides the full patient records, the specific clinics may suffer from information overload, which potentially prevents clinicians from isolating the prioritized information quickly. 

HIE allows two or more unaffiliated organizations to share patient EHR instantly. However, the information shared within the organization may not be accessible, especially after a merging or acquisition between providers using different EHR systems, causing information gaps (O’Malley, 2011). It is suggested that both intra- and inter-organizational HIE should be well established so that the completeness of patient profile could be fundamentally improved. However, such a requirement may create more complications for HIE structure designers and policymakers. For instance, it requires that the information senders and receivers have similar structures and levels of EHR systems. Yet, it is not ideal for the HIE between rural healthcare organizations and urban organizations. For instance, small and independent practice in rural areas may not have as many features and updates as those in urban organizations. What’s worse, some of the rural practices may not even have the HIT infrastructure or connective channels adequately built up for HIE. Such an issue of heterogeneity needs effort from all healthcare stakeholders.

The electronic access to information from external settings requires a push and pull exchange. When the information source and the patient information gap are both known, a push exchange could be initiated to deliver information to the target recipient. Yet when the information source for the information gap is unknown, the healthcare team needs to investigate for the information about the patient case externally, with a pull exchange as a form of a retrieval request. To govern the information exchange, three types of organization networks were developed as HIE initiatives: enterprise HIE network, EHR vendor HIE network, and community network (equivalent to the aforementioned RHIOs). Enterprise HIE networks are established among the dictated involved organizations to support patient care whiles facilitating the restriction of strategy and proprietary interests of the specific organizations. EHR vendor HIE networks are established for information exchange within a user community or a group of provider organizations utilizing an EHR system from the same vendor (Holmgren, 2017). The community HIE networks are established for the state-leveled organizations within the community that is usually funded by the HITECH grants. Thus sometimes, the participants of community HIE may encounter geographical obstacles.

To solve the problem of fragmented care, policymakers, researchers, industry groups, and healthcare professionals should work together on HIE to enhance the availability and accessibility of complete patient profiles. Meanwhile, patients’ support for HIE is also critical for its success because their consent is required for sharing their health information, and they are also an irreplaceable stakeholder for HIE (Esmaelzadeh, 2017). The patients’ attitudes towards sharing their personal health information can affect the design of future health information systems (Whiddett, 2006). Although many patients perceived HIE's benefit for a better quality of healthcare, their primary concern is the privacy and confidentiality issue: HIE is making EHR breaching even more harmful. Without the satisfactory confidentiality of EHR, we may expect a decreased participation of patients in HIE.  

## Benefits and Barriers to HIE

**Benefits to HIE**

Recipients of HIE's potential benefits include patients, providers, and the government/public health sector. The stakeholders group the potential benefits listed below.
1) __Patients__: Improve patient safety, clinical outcomes, and quality and satisfaction of care. Reduce unnecessary and redundant tests. Reduce medical errors. Promote patients’ participation in their health data.
2) __Providers__: Reduce costs and unnecessary paperwork. Improve access to data, monitoring of the health of patients, and clinical analytics opportunities. 
3) __Government/public health sector__: Improve in public health data access, reporting, and monitoring.

**Barriers to HIE**

Barriers to HIE listed below are categorized into barriers to HIE use and barriers to the implementation and sustainability of HIE.

_Barriers to HIE use_
1) __Lack of critical mass__: Patients do not share their data due to concerns about privacy and security, and when they do, there are issues with matching information to existing data, of which match rate can be high or low in different settings. When incomplete information is encountered frequently by providers, they stop using the system.
2) __Inefficient workflow__: Logins with too many clicks and policies forbidding proxy users make access to data hard and take more time. When a new workflow is to be implemented, there should also be educational training and technical support on use as well as culture change.
3) __Poorly designed interface and features__: The interface containing too much information made the users spend time scrolling for necessary information and going back and forth between pages, and the information was not filtered to provide meaningful information for users. Information was not timely updated as the hospital system, making the users to instead access the data from the hospital directly. Slowed response as more data was exchanged was also a problem.

_Barriers to implementation and sustainability of HIE_
1) __External Policy__: Though having the Federal and State laws in place contribute to the implementation, changes in policy, uncertainty on timelines of funding, and not every initiative gets funded are issues needing to be addressed. There are also mismatches between government goals and realities in implementing.
2) __Disincentives__: Cost is a barrier, and HIE investment does not lead to benefits for the hospital when redundant tests are reduced. Insufficient resources and concerns with data misuse and privacy are also disincentives for implementation.
3) __Structural characteristics__: Hospitals already in large systems have the data needed in their own system, and are less in need of external data exchange, thus lowering their use of HIE. If hospitals have service across states with different HIE, there can be a mismatch of HIE coverage and service for patients.
4) __Technology__: Differences in standards across organizations, interoperability across systems, system capability, system reliability, and difficulty in exchange cause by EHR interface are all technology barriers to HIE. 
5) __Lack of necessary components__: Providers, especially physicians, do not see its value and are not participating in the exchange.
6) __Fit__: Organizations not integrating HIE into their workflow either because they are incapable of doing it or reluctant to do it, maybe due to some unmet expectations, such as completeness and timeliness of data.

## References

Adler-Milstein J, Bates DW, Jha AK. Operational health information ex- changes show substantial growth, but long-term funding remains a concern. Health Aff (Millwood). 2013;32:1486-92. [PMID: 23840051] doi:10.1377 /hlthaff.2013.0124 

Agency for Healthcare Research and Quality. (2015). Health Information Exchange Evidence Report/Technology Assessment (Number 220) [PDF file]. Retrieved from https://effectivehealthcare.ahrq.gov/sites/default/files/pdf/health-information-exchange_research.pdf

Blumenthal D. Launching HITECH. N Engl J Med. 2010;362(5):382-385.
Department of Health Care Finance. (n.d.). Benefits of Health Information Exchange. Retrieved from https://dhcf.dc.gov/page/benefits-health-information-exchange

Esmaeilzadeh, P., & Sambasivan, M. (2016). Health Information Exchange (HIE): A literature review, assimilation pattern and a proposed classification for a new policy approach. Journal Of Biomedical Informatics, 64, 74-86. https://doi.org/10.1016/j.jbi.2016.09.011

Esmaeilzadeh, P., & Sambasivan, M. (2017). Patients’ support for health information exchange: a literature review and classification of key factors. BMC Medical Informatics And Decision Making, 17(1). https://doi.org/10.1186/s12911-017-0436-2

Holmgren, A., & Adler-Milstein, J. (2017). Health Information Exchange in U.S. Hospitals: The Current Landscape and a Path to Improved Information Sharing. Journal Of Hospital Medicine, 12(03), 193-198. https://doi.org/10.12788/jhm.2704

Hripcsak G, Kaushal R, Johnson KB, Ash JS, Bates DW, Block R, et al. The United Hospital Fund meeting on evaluating health information exchange. J Biomed Inform. 2007;40:S3-10. [PMID: 17919986] 

J.L. Kwan, L. Lo, M. Sampson, K.G. Shojania, Medication reconciliation during transitions of care as a patient safety strategy: a systematic review, Ann. Inter. Med. 158 (5) (2013) 397–403. 

Mathematica Policy Research and Harvard School of Public Health. DesRoches CM, Painter MW, Jha AK, eds. Health Information Technology in the United States, 2015: Transition to a Post-HITECH World (Executive Summary). September 18, 2015. Princeton, NJ: Robert Wood Johnson Foundation; 2015. 

Miller, R.H. & Miller, B.S. (2007). The Santa Barbara County Care Data Exchange: What Happened? _Health Affairs_, 26(2). doi: 10.1377/hlthaff.26.5.w568

O’Malley AS, Anglin G, Bond AM, Cunningham PJ, Stark LB, Yee T. Greenville & Spartanburg: Surging Hospital Employment of Physicians Poses Opportunities and Challenges. Washington, DC: Center for Studying Health System Change (HSC); February 2011. 6. 

S. Rosenfeld, S. Koss, K. Caruth, G. Fuller, Evolution of state health information exchange/a study of vision, strategy, and progress, AHRQ Publication, 2006 (06- 0057). 
 
The Office of the National Coordinator for Health Information Technology. (2017). HIE Benefits. Retrieved from https://www.healthit.gov/topic/health-it-basics/hie-benefits

Vest, J.R. & Gamm, L.D. (2010). Health information exchange: persistent challenges and new strategies. _J Am Med Inform Assoc_, 17(3):288-294. doi: 10.1136/jamia.2010.003673
Walker J, Pan E, Johnston D, Adler-Milstein J, Bates DW, Middleton B. The value of health care information exchange and interoperability. Health Aff (Millwood). 2005;Suppl Web Exclusives:W5-10-W5-18. [PMID: 15659453] 

Whiddett R, Hunter I, Engelbrecht J, Handy J. Patients’ attitudes towards sharing their health information. Int J Med Inform. 2006;75(7):530–41.

**Comment by pristineliving** [comment]
## Reflection by Abby

As Mu has already touched on, we're really starting to see all the different elements of what we've discussed come together – change management, as Mu says, but also PCMEs, security and privacy, EHRs, and of course the foundation: the healthcare system itself. It's been interesting to see how all these topics influence and impact each other; HIE as much affects any of these other areas as they do HIE. 

I've talked about in previous reflections some of my frustrations when it comes to addressing problems within the healthcare space that are not unique to the industry – that we are in many cases 'reinventing the wheel'. I think on some level this still holds true for HIE, but we're beginning to step more into health-specific issues. I actually think that some of the problems we see in information exchange comes from the way we approach the preceding problem. Especially from taking a closer look at the progression of HIE as both a process and a system, I'm beginning to see a pattern of issue-badaid-issue-bandaid.

In other words, it feels a little bit as though an issue, challenge, or even just potential concern will be raised and if it gets enough focus it will be addressed – whether through organizational, financial, regulatory, or legal means. But many of these 'fixes' don't look at the underlying core issue that's causing these surface-level symptoms. It's likely something that can't be fixed with a new piece of legislature, and as we've seen sometimes just the result of an idea running ahead of the technology available. But it's certainly a phenomenon I've observed.

**Comment by pristineliving** [comment]
## Reflection by Mu

I think implementing HIE corresponds to the section of health change management earlier. I actually felt health change management is an important topic and should come in earlier chapter, because we can see the theme repeating in the implementation of EHR and HIE. When there is change, barriers are bound to be there. HIE adds another layer of complexity for differences in systems, and therefore need even more coordination between organizations and technology support. When coordination is addressed, a lot of the barriers for implementation are gone, but it is not an easy task, and therefore the issues are still present as we’ve seen it.

**Comment by pristineliving** [comment]
## Reflection by Tianran

As a statistician, HIE is an exciting artifact for me because it would provide clinicians with a better-rounded patient profile and provide researchers with complete patient data. However, I feel that rural areas may encounter steep healthcare data quality discrepancies during the development of HIE. It is like a snowballing process: organizations with better EHR pipelines are likely to play together and evolve faster and faster. The rest with inadequately developed EHR infrastructure might be left behind for ages. The under-developed organizations may not only have information gaps: they may be an entirely blank chapter for the HIE infrastructure. On the other hand, I feel that the confidentiality issue is again haunting us. If the EHR is not secure, why bother sharing more patient information? Once a data breach occurs in an HIE-competent network, the consequence could be catastrophic. I still remember how shocked I was when my friend told me that the EPIC just got hacked in earlier this year, and millions of patient profiles are no longer secure or private. It is like an old dilemma appeared in a new story: confidentiality and EHR development, in a complication of HIE, never faded.
